src/patterns/filtering/single-selection.stories.ts

```diff
@@ -0,0 +1,146 @@
+import { Component, OnInit, OnDestroy } from "@angular/core";
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs } from "@storybook/addon-knobs/angular";
+import { TableModule, TableModel, TableHeaderItem, TableItem } from "../../table/table.module";
+import { DropdownModule } from "../../dropdown/dropdown.module";
+import { GridModule } from "../../grid/grid.module";
+import { UIShellModule } from "../../ui-shell/ui-shell.module";
+import { Carbon32Module } from "@carbon/icons-angular/lib/carbon/32";
+
+@Component({
+    selector: "app-sample-single-selection",
+    template: `
+    <div ibmGrid>
+        <div ibmRow class="header">
+            <ibm-header [brand]="brandTemplate">
+                <ibm-hamburger></ibm-hamburger>
+            </ibm-header>
+        </div>
+        <div ibmRow>
+            <div ibmCol [columnNumbers]="{'lg':3, 'md':3, 'sm':3}">
+                <label ibmText>
+                    Filter by:
+                    <ibm-dropdown placeholder="Type" inline="true" (selected)="onSelected($event)">
```

Comment:
Make sure there's a way to reset this filter

Reply:
done!